metamorph-kafka-010/src/main/java/net/researchgate/kafka/metamorph/kafka010/PartitionConsumerProvider.java

```diff
@@ -0,0 +1,19 @@
+package net.researchgate.kafka.metamorph.kafka010;
+
+import net.researchgate.kafka.metamorph.PartitionConsumer;
+import org.apache.kafka.common.serialization.Deserializer;
+
+import java.util.Properties;
+
+public class PartitionConsumerProvider implements net.researchgate.kafka.metamorph.PartitionConsumerProvider {
+
+    public <K, V> PartitionConsumer<K, V> createConsumer(Properties properties, Object keyDeserializer, Object valueDeserializer) {
+        // TODO: check instances
```

Comment:
I think we should be more specific about this comment and clearly state what feature is missing